sdk/appconfiguration/app-configuration/src/models.ts

```diff
@@ -163,8 +163,7 @@ export interface AddConfigurationSettingResponse
  * Response from deleting a ConfigurationSetting.
  */
 export interface DeleteConfigurationSettingResponse
-  extends ConfigurationSetting,
-    SyncTokenHeaderField,
+  extends SyncTokenHeaderField,
```

Comment:
Do we still expect a SyncToken header when deleting the configuration?

Reply:
It's totally possible via the protocol so, yes, although sync token usage is a pretty advanced case. Given a sync token, you can maintain consistency with queries or changes on data even in the face of fast-moving changes.

With that it seems possible to try to coordinate a scenario using the sync token with delete.